uberpoet/blazeprojectgen.py

```diff
@@ -108,16 +109,19 @@ def gen_app(self, app_node, node_list, target_loc, loc_json_file_path):
         if loc_json_file_path:
             loc_reader = locreader.LocFileReader()
             loc_reader.read_loc_file(loc_json_file_path)
-            module_index = {
-                n.name: self.gen_lib_module(n, loc_reader.loc_for_module(n.name)) for n in library_node_list
-            }
+            module_index = {}
```

Comment:
You can actually embed dictionary comprehensions into other dictionary / list comprehensions, and it would execute faster than this.  But I understand if you don't want to do it for readability reasons.

Reply:
done!